content/docs/get-started/gcp/deploy-stack.md

```diff
@@ -21,16 +21,20 @@ $ pulumi up
 This command instructs Pulumi to determine the resources needed to create the stack. First, a preview is shown of the changes that will be made:
 
 ```
+$ pulumi up
 Previewing update (dev):
+     Type                   Name               Plan       Info
```

Comment:
This looks like the code to modify (from deploy-changes.md), not the initial preview. Did you mean to make this change here?

Reply:
Good catch!